.github/workflows/ci.yml

```diff
@@ -48,3 +55,31 @@ jobs:
       - name: Check TinyGo version number consistency
         run: |
           builder/docker/tinygo/sanity.sh
+
+  # only run if reference is a tag
+  release:
+    if: startsWith(github.ref, 'refs/tags/v')
+
+    needs: [lint, test]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+
+      - uses: tibdex/github-app-token@v1
+        id: generate_token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
```

Comment:
Uses a GitHub application to generate single use GH tokens that can be used to do work across repos (as the GH_TOKEN cannot by default)